services/web/server/docker/boot.sh

```diff
@@ -63,5 +64,6 @@ else
     --workers="${WEBSERVER_GUNICORN_WORKERS:-1}" \
     --name="webserver_$(hostname)_$(date +'%Y-%m-%d_%T')_$$" \
     --access-logfile='-' \
-    --access-logformat='%a %t "%r" %s %b [%Dus] "%{Referer}i" "%{User-Agent}i"'
+    --access-logformat='%a %t "%r" %s %b [%Dus] "%{Referer}i" "%{User-Agent}i"' \
+    --worker-tmp-dir=/dev/shm
```

Comment:
can you elaborate a bit on why using `/dev/shm` ant not e.g. `/tmp` for the boot script. Or simply drop a comment here thx :-)

Reply:
https://pythonspeed.com/articles/gunicorn-in-docker/